.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Cache PureScript dependencies
         uses: actions/cache@v2
         with:
-          key: ${{ runner.os }}-spago-${{ hashFiles('**/*.dhall') }}
+          key: ${{ runner.os }}-spago-${{ hashFiles('**/*.dhall') }}-2
```

Comment:
What's this change about?

Reply:
It busted CI cache, so that it would get the `master` version of `control` where `<|>` is right-associative. Without it, it uses the old cache where `<|>` is still left-associative.